code/modules/spells/targeted/heal.dm

```diff
@@ -0,0 +1,89 @@
+/spell/targeted/heal
+	name = "Heal Other"
+	desc = "Mends basic wounds in the target."
+	abbreviation = "HL"
+	user_type = USER_TYPE_WIZARD
+	specialization = UTILITY
+	spell_levels = list(Sp_SPEED = 0, Sp_POWER = 0, Sp_RANGE = 0)
+
+	school = "transmutation"
+	charge_max = 300
+	cooldown_reduc = 75
+	cooldown_min = 150
+	invocation = "DI TIUB SEEL IM"
+	invocation_type = SpI_SHOUT
+	message = "<span class='danger'>You feel refreshed.<span>"
```

Comment:
span class 'good', 'info', or 'sinister' seems a better fit

Reply:
Good catch. I changed it to sinister.